docs/source/creating_an_image_processing_pipeline/index.rst

```diff
@@ -15,15 +15,16 @@ The :ref:`data model<data_model>`
 This part of the tutorial goes into more detail about why each of the stages in the example are
 needed, and provides some alternative approaches that can be used to build similar pipelines.
 
-The core functionalities of starfish pipelines are the detection (and decoding) of spots, and the
-segmentation of cells. Each of the other approaches are designed to address various characteristics
-of the imaging system, or the optical characteristics of the tissue sample being measured, which
-might bias the resulting spot calling, decoding, or cell segmentation decisions. Not all parts of
-image processing are always needed; some are dependent on the specific characteristics of the
-tissues. In addition, not all components are always found in the same order. *Starfish* is flexible
-enough to omit some pipeline stages or disorder them, but the typical order might match the
-following. The links show how and when to use each component of *starfish*, and the final section
-demonstrates putting together a "pipeline recipe" and running it on an experiment.
+The core functionalities of starfish pipelines are the detection (and :term:`decoding<Decoding>`)
```

Comment:
this links it to the right page?  very cool.

Reply:
not only the right page, it takes you directly to the glossary entry :)